nucliadb/nucliadb/common/cluster/settings.py

```diff
@@ -18,24 +18,23 @@
 # along with this program. If not, see <http://www.gnu.org/licenses/>.
 #
 import enum
-from typing import Optional
 
 from pydantic import BaseSettings, Field
 
 
 class ClusterDiscoveryMode(str, enum.Enum):
     MANUAL = "manual"
     KUBERNETES = "kubernetes"
+    SINGLE_NODE = "single_node"
 
 
 class Settings(BaseSettings):
     data_path: str = "./data/node"
     standalone_mode: bool = False
+    standalone_node_port: int = 10009
```

Comment:
NIT: you could add pydantic `Field` annotations so that we get descriptions on settings fields